validator/src/commands/staked_nodes_overrides/mod.rs

```diff
@@ -21,13 +21,8 @@ pub fn command(_default_args: &DefaultArgs) -> App<'_, '_> {
         )
 }
 
-pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
-    if !matches.is_present("path") {
-        println!("staked-nodes-overrides requires argument of location of the configuration");
-        exit(1);
-    }
-
-    let path = matches.value_of("path").unwrap();
+pub fn execute(matches: &ArgMatches, ledger_path: &Path) -> Result<(), String> {
+    let path = matches.value_of("path").expect("path is required");
```

Comment:
This argument is marked "required" in the `Arg` and the previous approach was needlessly repetitive